packages/table/components/Table.tsx

```diff
@@ -61,6 +63,7 @@ export class Table<T> extends React.PureComponent<ITableProps, {}> {
     return (
       <AutoSizer
         __dataThatTriggersARerenderWhenChanged={this.props.data} // passed to notify react-virtualized about updates
+        className={tableCss}
```

Comment:
Couldn’t we attach the table css rather to the multi grid instead of the autosizer?